### DISCUSSION TITLE
Does spatial_vector stack first element on top?

### DISCUSSION THREAD

**Discussion by cadop** [NVIDIA/warp#16] [Q&A] [answered]
In the sim example for `integrate_bodies` , the call to `wp.spatial_vector` has `(angular,linear)`.  In the utils https://nvidia.github.io/warp/_build/html/modules/functions.html#warp.spatial_top there is a method to get top and bottom.  So is getting top the angular velocity?

**Comment by mmacklin** [comment, marked as answer]
Yes that's right, this is a convention from Featherstone, but it is slightly non-intuitive when combined with a maximal coordinate rigid body solver so we may swap these around in a future version.
